Sources/VaporToolbox/PrepareAdd.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import Console
+
+public final class PrepareAdd: Command {
+    public let id = "add"
+
+    public let signature: [Argument] = []
+
+    public let help: [String] = [
+        "Create new database preparation file."
+    ]
+
+    public let console: ConsoleProtocol
+
+    public init(console: ConsoleProtocol) {
+        self.console = console
+    }
+
+    public func run(arguments: [String]) throws {
+        if arguments.count < 2 {
+            throw ToolboxError.general("Missing preparation name. Usage: vapor prepare add MyFirstPreparation")
```

Comment:
If you add the argument to the `public let signature: [Argument] = []` it will do this verification for you.